src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
The only issue I have with calling it update is that this is something that's going to be keyed off of the ```requestAnimationFrame``` loop. I'm not sure if anything other than animations really qualify to be modified in this particular place.

Thoughts on that?